connector/connect/client/jvm/pom.xml

```diff
@@ -73,7 +73,7 @@
     </dependency>
     <dependency>
       <groupId>com.lihaoyi</groupId>
-      <artifactId>ammonite_${scala.version}</artifactId>
+      <artifactId>ammonite_2.13.13</artifactId>
```

Comment:
Only for test
After the `Ammonite` supports `scala 2.13.14`, I will restore it.

Reply:
Yes, it's merged ten minutes ago. Looking forward to seeing the new release.